From: Colin Ian King <colin.i.king@gmail.com>
Date: Thu, 3 Jul 2025 11:22:19 +0100
Subject: net/mlx5: Fix spelling mistake "disabliing" -> "disabling"
Patch-mainline: v6.17-rc1
Git-commit: 0e86f3eb83c0d90ec082ceac925a500ec6ad604e
References: jsc#PED-14197 jsc#PED-14199 jsc#PED-15315

There is a spelling mistake in a NL_SET_ERR_MSG_MOD message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Reviewed-by: Mark Bloch <mbloch@nvidia.com>
Link: https://patch.msgid.link/20250703102219.1248399-1-colin.i.king@gmail.com
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/mellanox/mlx5/core/esw/qos.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/mellanox/mlx5/core/esw/qos.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/esw/qos.c
@@ -1353,7 +1353,7 @@ static int esw_qos_switch_tc_arbiter_nod
 					     &node->ix);
 	if (err) {
 		NL_SET_ERR_MSG_MOD(extack,
-				   "Failed to create scheduling element for vports node when disabliing vports TC QoS");
+				   "Failed to create scheduling element for vports node when disabling vports TC QoS");
 		return err;
 	}
 
